### DISCUSSION TITLE
Improving the error message

### DISCUSSION THREAD

**Discussion by sanzeeb3** [PHPMailer/PHPMailer#2562] [General]
When sending an email in WordPress with:

`wp_mail( 'nameSurname.@gmail.com', 'Subject', 'Message' );`

we get the error message: "You must provide at least one recipient email address". 

Though the recipient is invalid, it's provided. Can this message be improved in PHPMailer?

**Comment by Synchro** [comment]
The problem here is that the `addAddress` method returns `false` if an address is rejected, but Wordpress is not checking this, meaning that it continues to the point where it tries to send, and then runs into the problem you describe. PHPMailer can't tell why there are no addresses, so the fact that there are none is all it can report. I recommend reporting this error to WordPress since the problem is ow they are using PHPMailer, not in PHPMailer itself.

A secondary observation is that gmail itself will accept addresses with trailing dots in the local part (as in your example), even though that is an RFC822 violation.

**Reply by sanzeeb3** [reply]
Thank you for the explanation. I've repored to WordPress: https://core.trac.wordpress.org/ticket/54465
